.npmrc

```diff
@@ -0,0 +1 @@
+v16.12.1
```

Comment:
No needed because the correct file is nvmrc.

Reply:
AAAAAa

Reply:
Working on it!